_layouts/default.html

```diff
@@ -0,0 +1,26 @@
+{% assign lvl = page.url | append:'X' | split:'/' | size %}
+{% capture root %}{% for i in (3..lvl) %}../{% endfor %}{% endcapture %}
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Wobscale</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="{{ root }}site.css">
```

Comment:
Should this be `{{ root }}/site.css`? (see also next line)

Reply:
`{{ root }}` is a number of `../`, so if it's non-empty it always ends in a slash.

I don't think an extra slash would hurt, but it's not needed either.

Reply:
Ah ok cool. In that case, there is an extra `/` a couple lines down!